scripts/install

```diff
@@ -54,8 +54,8 @@ else
         your hardware and the result of the command \"uname -m\"." 1
 fi
 
-# https://docs.gitlab.com/ee/install/requirements.html#unicorn-workers
-unicorn_worker_processes=$(($(nproc) + 1 ))
+# https://docs.gitlab.com/ce/install/requirements.html#unicorn-workers
+unicorn_worker_processes=$(python -c "print(int($(nproc) * 1.5 + 1))")
```

Comment:
Why do you need python to execute nproc ?

Reply:
Not to execute nproc, but to have a float, and a cast to int. Do you have an easier way to do it?

Reply:
But you actually execute nproc, but ok you're need is to have a float instead of an int.
That's right that bash does not do float, but only integer. For float, you can use `bc`.
```bc <<< "$(nproc) * 1.5 + 1"```
or
```echo "$(nproc) * 1.5 + 1" | bc```

Reply:
Thanks!

At the end, I still need an int, so I will use:

`bc <<< "($(nproc) * 1.5 + 1) / 1"`